formula1_race_analysis/driver_model.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass, field
+
+
+@dataclass(order=True)
+class Driver:
+    driver_id: str
+    driver_name: str
+    car_model: str
+    start_time: str | None = field(default=None, init=False, compare=False)
```

Comment:
I would argue, that start time end time race duration and position are not the qualities of driver.
That is something related to the race results. Driver can participate in multiple races and have different times and positions but id name is still the same. Car_model is subjective. Can be property and can not.,